tests/cypress/integration/admin.test.js

```diff
@@ -0,0 +1,24 @@
+describe("Admin can login and make sure plugin is activated", () => {
+	it("Can activate plugin if it is deactivated", () => {
+		cy.login();
+		cy.visit("/wp-admin/plugins.php");
+
+		cy.get( 'body' ).then( ( $body ) => {
+			if( $body.find( '#the-list tr[data-slug="simple-page-ordering"] .deactivate > a' ).length > 0 ) {
```

Comment:
Are we following 'no space' before and after the `body/$body` and inside the `if` condition? OR are we following one space on both sides? Looking at the rest of the code, looks like we do not leave spaces.